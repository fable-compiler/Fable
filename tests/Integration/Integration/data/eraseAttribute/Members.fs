module EraseAttributeMembers

open Fable.Core

type input() =

    [<Erase>]
    member this.onChange
        with set (_: obj -> unit) = ()


type input with

    [<Erase>]
    member this.onChange2 set (_: obj -> unit) = ()
