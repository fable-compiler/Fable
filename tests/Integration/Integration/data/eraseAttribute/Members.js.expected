import { class_type } from "./fable_modules/fable-library-js.4.22.0/Reflection.js";

export class input {
    constructor() {
    }
}

export function input_$reflection() {
    return class_type("EraseAttributeMembers.input", undefined, input);
}

export function input_$ctor() {
    return new input();
}

